src/nrc/templates/components/pagination/pagination.html

```diff
@@ -1,33 +0,0 @@
-{% if is_paginated %}
-<nav aria-label="Page navigation example">
```

Comment:
Dit bestand is nog wel nodig zie ik (o.a. [hier](https://github.com/VNG-Realisatie/notificaties-api/blob/master/src/nrc/logviewer/templates/notifications/logviewer/notificatie-list.html#L49)).